services/bots/src/github-webhook/handlers/review_drafter.ts

```diff
@@ -0,0 +1,81 @@
+import {
+  PullRequestReadyForReviewEvent,
+  PullRequestReviewSubmittedEvent,
+} from '@octokit/webhooks-types';
+import { EventType, HomeAssistantRepository } from '../github-webhook.const';
+import { WebhookContext } from '../github-webhook.model';
+import { BaseWebhookHandler } from './base';
+
+const MESSAGE_ID = '<!-- ReviewDrafterComment -->';
+
+const REVIEW_COMMENT = `${MESSAGE_ID}
+Please take a look at the requested changes, and use the **Ready for review** button when you are done, thanks :+1:
```

Comment:
I would explain a bit more what happened.

> Your pull request has been marked as "request for changes" and has automatically been marked as draft. Please address the comments and use the **Ready for review** button when you are done, thanks :+1:

Reply:
That's the goal of the linked blogpost?
This (repeating) comment should IMO be short and link to more information.

Reply:
Yeah agree, in longer term we could replace the URL with a more extensive/dedicated page even (that covers more subjects). But keeping the comment short and to the point is probably not a bad idea. (For the same reason we don't let netlify or codecov post comments).

Reply:
ok 👍